modules/extensions-support/src/test/java/org/mule/module/extension/internal/metadata/extension/MetadataOperations.java

```diff
@@ -111,6 +112,26 @@ public Object outputMetadataWithoutKeysWithKeyParam(@Connection MetadataConnecti
         return null;
     }
 
+    // Resolver for output and content
+    // Without keysResolver
+    // With KeyParam
+    // Uses Cache
+    @MetadataScope(outputResolver = TestResolverWithCache.class, contentResolver = TestResolverWithCache.class)
+    public Object contentAndOutputCacheResolver(@Connection MetadataConnection connection, @MetadataKeyParam String type, @Content Object content)
+    {
+        return null;
+    }
+
+    // Resolver for content
```

Comment:
We should replace this multi line comments with a more descriptive (or at least, better formatted) javadoc, but this has to be at method level.
The idea is to communicate quickly which of the multiple combinations for Content, Output, KeyParam & Resolvers is being used by this method. Even when the Resolver's name should communicate part of this, and the signature completes the idea, is easier to identify the case during testing